### DISCUSSION TITLE
Any way to install a VPN on school chromebook without the playstore or webstore?

### DISCUSSION THREAD

**Discussion by CALR3F** [3kh0/ext-remover#515] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
A VPN would allow me to bypass some filters.

**Comment by obkduishinijf** [comment]
probably not, then again i just use sh1mmer to unenroll so.. 🤷

**Reply by fnfgo** [reply]
> probably not, then again i just use sh1mmer to unenroll so.. 🤷

yes there is stop copying everything i say dumbass

**Reply by obkduishinijf** [reply]
> > probably not, then again i just use sh1mmer to unenroll so.. 🤷
> 
> yes there is stop copying everything i say dumbass

shut the fuck up, you think that everyone cares about your stupid fucking opinion, it's just us stating the same thing, you crusty dusty musty broken ugly silly goofy retarded ass, get off the internet and get some bitches.

**Reply by pl4udible** [reply]
will you shut up, brudda 
this is a discussion not a beefing house

**Reply by obkduishinijf** [reply]
> will you shut up, brudda this is a discussion not a beefing house

tell him that, ugly mf got mad over us saying the same thing

**Reply by fnfgo** [reply]
> will you shut up, brudda this is a discussion not a beefing house

who invited you?

**Comment by funkymokey** [comment, marked as answer]
There actually is, using onc files on the Chrome://network page you can get a VPN, its just that i don't know where to get one.

**Reply by Aka-but-nice** [reply]
> There actually is, using onc files on the Chrome: //network page you can get a VPN, its I don't know where to get one.

No, onc files can't precisely "mimic" a VPN. They can only block specific requests, this is very limited. Now, I recommend, whoever is reading this at least try it, even though my answer stated it wouldn't.
